letsencrypt/views/base_config_settings.xml

```diff
@@ -6,15 +6,25 @@
         <field name="inherit_id" ref="base_setup.view_general_configuration" />
         <field name="arch" type="xml">
             <xpath expr="./group[last()]" position="after">
-                <field name="letsencrypt_needs_dns_provider" invisible="1" />
-                <group
-                    name="letsencrypt"
-                    string="LetsEncrypt"
-                    attrs="{'required':
-                        [('letsencrypt_needs_dns_provider', '=', True)]}">
+                <group name="letsencrypt">
+                    <field
+                        name="letsencrypt_needs_dns_provider"
+                        invisible="1" />
                     <field name="letsencrypt_dns_provider" />
-                    <field name="letsencrypt_dns_login" />
-                    <field name="letsencrypt_dns_key" />
+                    <group
+                        name="letsencrypt"
+                        string="LetsEncrypt"
+                        attrs="{
+                            'required':[
+                                ('letsencrypt_needs_dns_provider', '=', True)
+                            ],
```

Comment:
can a group be required by now? I fear you need to add this to the single fields